addons/sourcemod/scripting/mapchooser_extended.sp

```diff
@@ -897,12 +888,21 @@ public Action Event_PreRoundEnd(Handle event, const char[] name, bool dontBroadc
 
 			ClearSyncHud(i, g_hHud);
 			ShowSyncHudText(i, g_hHud, "Next Map: %s", nextMap);
+			CPrintToChat(i, "{lightgreen}Next Map: {green}%s", nextMap);
 		}
 	}
+	else
+	{	
+		CPrintToChatAll("{lightgreen}Next Map: {green}%s", nextMap);	
+		CPrintToChatAll("{lightgreen}Next Map: {green}%s", nextMap);	
+		CPrintToChatAll("{lightgreen}Next Map: {green}%s", nextMap);	
+	}
 
-	return Plugin_Handled;
+	if(g_hHud != INVALID_HANDLE)
```

Comment:
you can put this delete at the end of the condition right before in `if (g_hHud != INVALID_HANDLE)`